lib/build/source-inclusion.js

```diff
@@ -27,8 +28,16 @@ exports.SourceInclusion = class {
     this.items.push(item);
   }
 
+  isExcluded(item){
+	  let found = this.excludes.findIndex(x => {			
+			let exclusion = new this.bundle.Minimatch(x);
```

Comment:
Yes, that's basically the approach I was thinking of. Give it a shot and if it works, amend this PR and I'll get it merged and released this weekend.